R/zchunk_batch_modeltime.xml.R

```diff
@@ -0,0 +1,36 @@
+#' module_modeltime_batch_modeltime.xml
```

Comment:
Although I'm normally in favor of using `.` in place of `_`, I'm not a huge fan of naming files such that they appear to have two file extensions, such as ".xml.R" in this file.  In a UI that allows you to hide file extensions (itself not a great idea, but also rather common), this will appear as `zchunk_batch_modeltime.xml`, which looks makes it look like a stray XML file.

Maybe it's not really a big deal, but it seems like a potential source of needless confusion.